Add unique id/class to indentify activate user UI

MartinRiese opened the pull request:
## Product Description

Add unique id to show deactivated/active web users. 
Add unique class to deactivate/reactivate web user buttons.

## Technical Summary

https://dimagi.atlassian.net/browse/USH-6075

## Feature Flag

DEACTIVATE_WEB_USERS

## Safety Assurance

Just adding html ids/classes that are non-functional

### Safety story
<!--
Describe how you became confident in this change, such as
local testing, why the change is inherently safe, and/or plans to limit the blast radius of a defect.

In particular consider how existing data may be impacted by this change.
-->

### Automated test coverage

None

### QA Plan

None

### Rollback instructions

<!--
If this PR follows standards of revertability, check the box below.
Otherwise replace it with detailed instructions or reasons a rollback is impossible.
-->

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

MartinRiese committed:
Add unique id/class to indentify activate user UI

Add unique id to show deactivated/active web users.
Add unique class to deactivate/reactivate web user buttons.

MartinRiese committed:
Log web user active stage change during import

esoergel commented on corehq/apps/user_importer/importer.py:
```diff
@@ -821,6 +824,11 @@ def _modify_existing_user_in_domain(self, membership, role_qualified_id,
 
         imported_is_active = cv['is_active_in_domain']
         if str(current_user.is_active_in_domain(self.domain)) != imported_is_active:
+            record_event(WebUserRow.USER_IMPORTER_USER_ENABLED_STATUS_CHANGED, self.importer.upload_user, {
```
Why make the name a constant rather than define it in-line?  Follow-up question - any performance concerns with doing this in bulk?  Like what if they inactivate a ton of users all at once?  Does `record_event` batch metric submissions?  Is there a way to do so?

esoergel commented on corehq/apps/user_importer/importer.py:
```diff
@@ -9,6 +9,7 @@
 from memoized import memoized
 from django.db import DEFAULT_DB_ALIAS
 
+from corehq.apps.analytics.tasks import record_event
```
:star:

MartinRiese committed:
Add unique id/class to indentify activate user UI

Add unique id to show deactivated/active mobile workers.
Add unique class to deactivate/reactivate mobile workers buttons.

MartinRiese committed:
Only log overall import active change

Also log mobile worker active change
